packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -53,15 +55,33 @@ const $singleton = Symbol('singleton');
  * the texture.
  */
 export class Renderer extends EventDispatcher {
-  static[$singleton] = new Renderer({debug: isDebugMode()});
+  protected static[$singleton]: Renderer|null = null;
+  protected static[$singletonWithTransparency]: Renderer|null = null;
 
-  static get singleton() {
-    return this[$singleton];
+  static get singleton(): Renderer {
```

Comment:
It seems weird to call this a singleton now that we are creating up to two of them. Maybe we should just call them `opaqueRenderer` and `transparentRenderer`?

Reply:
ok, that's renamed.